### PR TITLE
QA: Logo padding, SEO title change

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Minor padding fixes
- Changing hyphen to em dash
- Allow for featured product layout setting to be used on all Collections and not just the Homepage collection 
### Type(s) of changes

- [x] Update to an existing feature

### How Has This Been Tested?

https://crugykd3mxeiqkq5-52674822324.shopifypreview.com

### CODE REVIEW

File: theme/snippets/meta.liquid [hunk 1]
```diff
@@ -16,32 +16,32 @@
...
+  <meta property="og:title" content="{{ 'seo.default_title' | t }} — {{ page.title | escape }}">
   <meta name="twitter:description" content="{{ page_description | escape }}">
   <meta property="og:image" content="https:{{ 'index-share-card.jpg' |  asset_url }}">
 
 {% else %}
-  <title>{{ 'seo.default_title' | t }} - {{ page.title | escape }}</title>
+  <title>{{ 'seo.default_title' | t }} — {{ page.title | escape }}</title>
   <meta property="og:type" content="website">
-  <meta property="og:title" content="{{ 'seo.default_title' | t }} - {{ page.title | escape }}">
+  <meta property="og:title" content="{{ 'seo.default_title' | t }} — {{ page.title | escape }}">
```
Comment (chrismekim) [inline theme/snippets/meta.liquid:44]: Changing to em dash

**Comment by chrismekim** [comment]
@sepowitz waiting for Jake to see the preview link and confirm a padding change on the logo :)

File: theme/templates/collection.liquid [hunk 2]
```diff
@@ -1,4 +1,5 @@
-{% assign show_featured_product = collections.frontpage.metafields.accentuate.show_featured_product %}
+{% assign show_featured_product = collection.metafields.accentuate.show_featured_product %}
```
Comment (chrismekim) [inline theme/templates/collection.liquid:1]: Was only checking for the frontpage collection previously!

**Review by sepowitz — APPROVED** [review]
LGTM! Not seeing the featured image outside the homepage, but assuming that's because the setting has not been set to true in accentuate?

**Comment by chrismekim** [comment]
@sepowitz exactly!

**Pull request merged**